components/icu/src/lib.rs

```diff
@@ -106,6 +106,10 @@
 #[doc(inline)]
 pub use icu_calendar as calendar;
 
+#[cfg(feature = "experimental")]
+#[doc(inline)]
+pub use icu_casemapping as casemapping;
```

Comment:
Nit: Make it clear in the docs somehow that this is an experimental crate that must be enabled with the "experimental" feature, and that the API may change at any time.